man/systemd.network.xml

```diff
@@ -247,6 +247,12 @@
         <listitem>
           <para> A boolean. Enables or disables the change the MULTICAST flag on the device.</para>
         </listitem>
+      </varlistentry>
+       <varlistentry>
+        <term><varname>AllMulticast=</varname></term>
+        <listitem>
+          <para> A boolean. Accept or reject all multicast frames.</para>
```

Comment:
Please make this explain what this setting does, and how it differs from Multicast= above. See https://serverfault.com/questions/321245/what-is-the-difference-between-multicast-and-allmulti-ifconfig-ifconfig-flags.